repo-management.html

```diff
@@ -136,17 +127,17 @@
     <h3>PR Details</h3>
     <p>
       If the PR is not in an acceptable state, this will list problematic users with a link to fix
-      them each. The fix can either be "Add to system" or "Edit" (details below).
+      them each, and alternatively a button to "Mark the PR as non-substantive".</p>
+      <p>For problematic users, the fix can either be  "Add to system" or "Edit" (details below).
     </p>
     <p>
-      The idea is that the vast majority of non-acceptable PRs in the first few weeks will come from
-      people who are simply not known, but that relatively quickly it ought to become a less
-      frequent occurrence.
+      The idea is that the vast majority of non-acceptable PRs for a newly added repo will come from
+      people who are simply not known.
     </p>
     <p>
       If it so happens that all of the problematic users can be added to the system or to the right
       group, and that you have done so, then you can return to the PR details page and hit
```

Comment:
s/so happens.*done so, then you can/all problematic users are added to the system and group,/